fvtest/porttest/omrdumpTest.cpp

```diff
@@ -176,6 +176,48 @@ TEST(PortDumpTest, dump_test_create_dump_with_NO_name)
 }
 
 
+/**
+ * Test omrdump_create(), this time passing in core file name,
+ * but with no path. It should get created in the current
+ * working directory.
+ * Verify that the returned core file name actually exists.
+ *
+ * @note this only tests IEATDUMP generation on z/OS
+ */
+TEST(PortDumpTest, dump_test_create_dump_with_no_path)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
+	const char *testName = "dump_test_create_dump_with_no_path";
+	uintptr_t rc = 99;
+	char coreFileName[EsMaxPath];
+
+	reportTestEntry(OMRPORTLIB, testName);
+
+	strcpy(coreFileName, testName);
+
+	portTestEnv->log("calling omrdump_create with filename: %s\n", coreFileName);
+#if !defined(J9ZOS390)
+	rc = omrdump_create(coreFileName, NULL, NULL);
+#else
```

Comment:
Ref: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif

```suggestion
#else /* !defined(J9ZOS390) */
```